src/bindings/jna/README.md

```diff
@@ -1,43 +1,121 @@
-A full java binding using JNA.
+A full java binding using JNA. This binding requires elektra to be installed on 
+the system to work.
 
 ## Usage ##
 
+To use the bindings in a java project, we have to include the jar file 
+libelektra-0.8.19.jar in the project. This jar is created upon build, if you 
```

Comment:
Yes i'm aware of that as far as i've seen there is no clean possibility to use the * inside code. I considered it to be easy to understand to use the appropriate version number as most developers know markdown anyway so i just did i that way. any better idea? If not, i think we can finally merge it.